pyproject.toml

```diff
@@ -34,18 +38,19 @@ alembic = "^1.4.2"
 python-jose = {extras = ["cryptography"], version = "^3.2.0"}
 passlib = {extras = ["bcrypt"], version = "^1.7.2"}
 python-multipart = "^0.0.5"
+coverage = {extras = ["toml"], version = "^5.3"}
```

Comment:
Is really necessary for this lib being in both configs (prod and test)?

Reply:
Sorry, i duplicate the lines, removing...